spring-boot-project/spring-boot-docs/src/docs/asciidoc/production-ready-features.adoc

```diff
@@ -1741,17 +1744,63 @@ To export metrics to {micrometer-registry-docs}/dynatrace[Dynatrace], your API t
 	        uri: "YOUR_URI"
 ----
 
-You can also change the interval at which metrics are sent to Dynatrace:
+===== API v2
+
+The API v2 can be used int two ways: If a local OneAgent is running on the machine, it is enough to tell the exporter to use v2 and metrics will be automatically exported to the https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/local-api/[local OneAgent endpoint], which forwards them to the Dynatrace backend:
 
 [source,yaml,indent=0,configprops,configblocks]
 ----
 	management:
 	  metrics:
 	    export:
 	      dynatrace:
-	        step: "30s"
+	        api-version: "v2"
 ----
 
+If no local OneAgent is running, the token and URI are required.
+The https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication/[API token] must have the `metrics.ingest` permission set.
+
+[source,yaml,indent=0,configprops,configblocks]
+----
+	management:
+	  metrics:
+	    export:
+	      dynatrace:
+	        api-version: "v2"
+	        api-token: "YOUR_TOKEN"
+	        uri: "YOUR_URI"
+----
+
+When using the Dynatrace v2 API, new optional features are available:
+
+* Metric key prefix: set a prefix that will be prepended to all exported metric names.
+* Enrich with OneAgent metadata: If a OneAgent is running, enrich metrics with information about the host and the process.
+* Default dimensions: Specify key-value pairs that are added to all exported metrics.
+
+[source,yaml,indent=0,configprops,configblocks]
+----
+	management:
+	  metrics:
+	    export:
+	      dynatrace:
+	      # ... token and uri or blank for OneAgent export, and api-version: "v2" ...
```

Comment:
Odd indentation and api-version should not be commented out.

Reply:
this is just to tell users that they should use `apiVersion: v2`. I didnt put it here explicitly to reduce duplication but it makes sense to add it. I added it back.